modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,178 @@
+
+* With the current release, the `EventListener` resource sends a response before the triggers finish processing. (link:https://github.com/tektoncd/triggers/pull/1132[#1132])
++
+[IMPORTANT]
+====
+Breaking change: With this change, the `EventListener` resource stops responding with a `201 Created` status code when it creates resources. Instead, it responds with a `202 Accepted` response code.
+====
+
+* The current release removes the `podTemplate` field from the `EventListener` resource. (link:https://github.com/tektoncd/triggers/pull/1118[#1118])
++
+[IMPORTANT]
+====
+Breaking change: The `podTemplate` field, which was deprecated as part of link:https://github.com/tektoncd/triggers/pull/1100[#1100], has been removed.
+====
+
+* The current release removes the deprecated `replicas` field from the specification for the `EventListener` resource. (link:https://github.com/tektoncd/triggers/pull/1113[#1113])
++
+[IMPORTANT]
+====
+Breaking change: The deprecated `replicas` field has been removed.
+====
+
+[id="deprecated-features-pipelines-operator-1-6_{context}"]
+=== {pipelines-title} Operator
+
+* In {pipelines-title} 1.6, the values of `HOME="/tekton/home"` and `workingDir="/workspace"` are removed from the `Steps` specification.
++
+Instead, {pipelines-title} sets `HOME` and `workingDir` to the values defined by your `Steps` containers. You can override these values in your `Steps` specification.
++
+To use the older behavior, you can change the `disable-working-directory-overwrite` and `disable-home-env-overwrite` fields in the `TektonConfig` CR to `false`:
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+  kind: TektonConfig
+  metadata:
+    name: config
+  spec:
+    pipeline:
+      disable-working-directory-overwrite: false
+      disable-home-env-overwrite: false
+  ...
+----
++
+[IMPORTANT]
+====
+The `disable-working-directory-overwrite` and `disable-home-env-overwrite` fields in the `TektonConfig` CR  are deprecated and are candidates for removal after nine months.
+====
++
+(link:https://issues.redhat.com/browse/SRVKP-1465[SRVKP-1465])
+
+[id="known-issues-1-6_{context}"]
+== Known issues
+
+* When you run Maven and Jib-Maven cluster tasks, the default container image is supported only on Intel (x86) architecture. Therefore, tasks will fail on IBM Power Systems (ppc64le), IBM Z, and LinuxONE (s390x) clusters. As a workaround, you can specify a custom image by setting the `MAVEN_IMAGE` parameter value to `maven:3.6.3-adoptopenjdk-11`.
+// issue # is unknown.
+
+* On IBM Power Systems, IBM Z, and LinuxONE, the `s2i-dotnet` cluster task is unsupported.
+// issue # is unknown.
+
+* Before you install `tekton` catalog tasks on IBM Power Systems (ppc64le), IBM Z, and LinuxONE(s390x) using `tkn hub`, verify if the task can be executed on these platforms. To check if `ppc64le` and `s390x` are listed in the "Platforms" section of the task information, you can run the following command: `tkn hub info task <name>`
```

Comment:
`tekton` is usually displayed as ordinary text

Reply:
Updated. Thanks.